Cargo.toml

```diff
@@ -33,6 +33,7 @@ default = []
 nightly = []
 thiscall-abi = ["nightly"]
 static-detour = ["nightly"]
+extra-impls = []
```

Comment:
Is this feature expected to be expanded to any other types of impl? If not, I'd expect a less generic feature name. Diesel, for instance has very specific features, even including the number in the name.

* `32-column-tables`
* `64-column-tables`
* `128-column-tables`